release-please-config.json

```diff
@@ -81,7 +84,8 @@
         "api",
         "shared",
         "hvm",
-        "terraform-provider"
+        "terraform-provider",
+        "hapi-proto"
```

Comment:
Do these need to be linked?

If so that's ok but we'll need to be careful with the initial release to ensure it doesn't bump happy cli/api/...etc up to v1.0.0. We might wanna work through that initial release together.

Reply:
Ok, I'll remove it til later.

Reply:
I guess if happy api is going to use the protos they should probably be linked 🤔